### ISSUE TITLE
Invert the Order it shows object details

### ISSUE THREAD

**Issue by 0x53A** [EamonNerbonne/ExpressionToCode#84]
If I have this Unit Test

```C#
        [Fact]
        public void Test_PAssert()
        {
            var value = new { A = 1, b = new { c = 2, d = "4", e = new { f = 5 } } };
            PAssert.That(() => value.b.e.f == 7);
        }
```

then it will print

```
ExpressionToCodeLib.AssertionFailedException : assertion failed

value.b.e.f == 7
      value   →   new {
              A = 1,
              b = new {
                  c = 2,
                  d = "4",
                  e = new {
                      f = 5,
                  },
              },
          }
    value.b   →   new {
              c = 2,
              d = "4",
              e = new {
                  f = 5,
              },
          }
  value.b.e   →   new {
              f = 5,
          }
value.b.e.f   →   5
```

This is absolutely awesome.

What do you think about reversing the order?
I think it may be easier to see the core issue by going from inner-most to outer-most:

```
ExpressionToCodeLib.AssertionFailedException : assertion failed

value.b.e.f == 7

value.b.e.f   →   5

  value.b.e   →   new {
              f = 5,
          }
    value.b   →   new {
              c = 2,
              d = "4",
              e = new {
                  f = 5,
              },
          }
      value   →   new {
              A = 1,
              b = new {
                  c = 2,
                  d = "4",
                  e = new {
                      f = 5,
                  },
              },
          }
```

**Comment by EamonNerbonne** [comment]
This was changed for 3.0.  It may not be an improvement everywhere, but you've got a few examples; might as well see how it turns out.

**Comment by EamonNerbonne** [comment]
Implemented in #85.
